diff --git a/meson.build b/meson.build
index b8d9a9b..ce2ae5f 100644
--- a/meson.build
+++ b/meson.build
@@ -12,10 +12,10 @@ dependency('gio-2.0', version: '>= 2.66')
 dependency('gtk4', version: '>= 4.0.0')
 dependency('libadwaita-1', version :'>=1.0.0')
 dependency('zbar', version: '>= 0.20')
-dependency('gstreamer-1.0', version: '>= 1.18')
-dependency('gstreamer-base-1.0', version: '>= 1.18')
-dependency('gstreamer-plugins-base-1.0', version: '>= 1.18')
-dependency('gstreamer-plugins-bad-1.0', version: '>= 1.18')
+dependency('gstreamer-1.0', version: '>= 1.16')
+dependency('gstreamer-base-1.0', version: '>= 1.16')
+dependency('gstreamer-plugins-base-1.0', version: '>= 1.16')
+dependency('gstreamer-plugins-bad-1.0', version: '>= 1.16')
 
 find_program('cargo', required: true)
 find_program('glib-compile-resources', required: true)
